explore/src/main/scala/explore/proposal/InviteUserPopup.scala

```diff
@@ -93,11 +99,17 @@ object InviteUserPopup:
                 FormInputTextView(
                   id = "email-invite".refined,
                   value = emailView,
-                  label = "email",
+                  label = "Email",
                   disabled = inviteState.get === CreateInviteProcess.Running,
                   validFormat = MailValidator.optional,
                   onValidChange = v => validEmail.setState(v)
-                )(^.autoComplete := "off")
+                )(^.autoComplete := "off"),
+                FormEnumDropdownOptionalView(
```

Comment:
I wonder if we can show the flags of each partner, It can be done later though

Reply:
Yeah that would be great. But I'm not sure how since a `Display` works with strings, not VDomElements

Reply:
@hugo-vrijswijk There's a `Render` typeclass defined in `lucuma-ui`. It's the equivalent of `Display` but returns a `VdomNode`.

Reply:
But `FormEnumDropdownOptionalView` takes a `Display`

Reply:
Shall we change that to a `Render` ?

Reply:
It feels like that would be a breaking change for a lot of places. Or is any Display a valid Render?

Reply:
It's not, but we can add an implicit conversion.